pagopa/io-app
# feat(IT Wallet): [SIW-1594] Add official Driving License credential assets

## Short description
This PR adds the official assets fro the Driving License credential. It also fixes some data visualization problems and adds the swipe to flip gestures for skeumorphic cards.

## List of changes proposed in this pull request
- Added `mdl_back.png` and `mdl_front.png` assets
- Removed "Digital version" badge from skeumorphic cards
- Moved `ItwDigitalVersionBadge` component to the common folder
- Added the ability to customize `CardLabel` base font size and font weight
- Extended `CardClaim` props to allow label customization (font size, font weight and format for dates)
- Adjusted card data position for MDL credential
- Added missing MDL claims on skeumorphic card (`issuing_authority`, `restrictions_conditions`)
- Adjusted skeumorphic card aspect ratio
- Added swipe gesture to `ItwPresentationCredentialCard` component

## How to test
Open the MDL credential details screen, check that the card and its data is correctly displayed. Try to flip the card using the swipe gesture.

## Preview

https://github.com/user-attachments/assets/647e956a-fe42-4688-9059-3f3b8fda211f
